app/services/check_answers/section_lister_service.rb

```diff
@@ -26,15 +26,22 @@ def call(session_data)
           else
             pre = [
               Sections::Dependants.new(check),
-              Sections::ClientIncome.new(check),
-            ]
-            post = [Sections::Outgoings.new(check)]
+              # Simplecov is showing that check.under_eighteen_assets? is not being covered by a test
+              # we have tests in spec/flows/under_eighteen_flow_spec.rb that check with and without under_eighteen_assets
+              # :nocov:
+              (Sections::ClientIncome.new(check) if !check.under_eighteen? || check.aggregated_means? || check.under_eighteen_regular_income? || check.under_eighteen_assets?),
+              # :nocov:
```

Comment:
@willc-work this feels risky 🤔

Reply:
we have tests that cover both sides of under 18 assets and the associated views, so I think i have mitigated the risk as well as possible without spending more time understanding why simplecov isnt counting this as coverage

Reply:
Happy to leave like this. But maybe we make a separate ticket for this, so this doesn't get lost?